tgui/packages/tgui/interfaces/Pacman.js

```diff
@@ -0,0 +1,174 @@
+                selected={active}
+                onClick={() => act('toggle_power')} />
+              )}>
+              <Flex direction="row">
+                <Flex.Item width="50%" className="ml-1">
+                  <Flex direction="row">
+                    <Flex.Item width="40%" align="center" color="label">
+                      Power setting:
+                    </Flex.Item>
+                    <Flex.Item width="20%">
+                      <Flex direction="column" align="center">
+                        <Flex.Item>
+                          <Knob
+                            value={output_set}
+                            minValue={1}
+                            maxValue={output_max}
+                            ranges={{ color: [1, output_max] }}
+                            onDrag={(e, value) => act('change_power', {
+                              change_power: value,
+                            })}
+                          />
+                        </Flex.Item>
+                        <Flex.Item>
+                          <NumberInput
+                            value={output_set}
+                            minValue={1}
+                            maxValue={output_max}
+                            step={1}
+                            className="mt-1"
+                            onDrag={(e, value) => act('change_power', {
+                              change_power: value,
+                            })}
+                          />
+                        </Flex.Item>
+                      </Flex>
+                    </Flex.Item>
+                    <Flex.Item width="40%" align="center">
+                      ({output_watts} W)
+                    </Flex.Item>
+                  </Flex>
+                </Flex.Item>
+                <Flex.Item width="50%">
+                  <LabeledList>
+                    <LabeledList.Item label="Temperature">
+                      <ProgressBar
+                        value={tmpRatio}
+                        ranges={{
+                          green: [-Infinity, 0.33],
+                          orange: [0.33, 0.66],
+                          red: [0.66, Infinity],
+                        }}>
+                        {tmp_current} &#8451;
+                      </ProgressBar>
+                    </LabeledList.Item>
+                    <LabeledList.Item label="Status">
+                      {tmp_overheat > 50 && (
+                        <Box color="red"> CRITICAL OVERHEAT! </Box>
+                      )}
+                      {tmp_overheat > 20 && tmp_overheat <= 50 && (
+                        <Box color="orange"> WARNING: Overheating! </Box>
+                      )}
+                      {tmp_overheat > 1 && tmp_overheat <= 20 && (
+                        <Box color="orange"> Temperature High </Box>
+                      )}
+                      {tmp_overheat === 0 && (
+                        <Box color="green"> Optimal </Box>
```

Comment:
Any reason for the spacing before and after the `<Box>` tags

Reply:
No, removed